root/static/scripts/common/i18n/semicolonOnlyList.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * @flow strict
+ * Copyright (C) 2015-2016 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+function _semicolonOnlyList<Input, Output>(
+  l: ExpandLFunc<Input, Output>,
+  items: $ReadOnlyArray<Input>,
+): Output | string {
+  const length = items.length;
+
+  if (!length) {
+    return '';
+  }
+
+  let output = l('{last_list_item}', {
+    last_list_item: items[length - 1],
+  });
+
+  for (let i = length - 2; i >= 0; i--) {
+    output = l('{semicolon_only_list_item}; {rest}', {
+      rest: output,
+      semicolon_only_list_item: items[i],
+    });
+  }
+
+  return output;
+}
+
+const semicolonOnlyList = (items: $ReadOnlyArray<VarSubstArg>) => (
+  _semicolonOnlyList<VarSubstArg, Expand2ReactOutput>(exp.l, items)
+);
+
+const semicolonOnlyListText = (items: $ReadOnlyArray<StrOrNum>) => (
+  _semicolonOnlyList<StrOrNum, string>(texp.l, items)
+);
+
+export default semicolonOnlyList;
+
+export {semicolonOnlyListText};
```

Comment:
Isn't `export const x` clearer? We use that much more often than `export {x}` at the end (but we do use it this way a few times, so i guess it's not a big deal).

Reply:
Both seem fine to me but I simplified it by moving the `export default` to the `semicolonOnlyList` declaration, and then just doing `export function semicolonOnlyListText(...)` for parity.